api/serializers.py

```diff
@@ -40,5 +40,5 @@ class Meta:
 class ProfileSerializer(serializers.ModelSerializer):
     class Meta:
         model = Profile
-        fields = ['id', 'server_storage', 'subdomain', 'onboarding_state']
-        read_only_fields = ['id', 'subdomain']
+        fields = ['id', 'server_storage', 'subdomain', 'onboarding_state', 'subscription_date']
+        read_only_fields = ['id', 'subdomain', 'subscription_date']
```

Comment:
Did this change in dev? Looks like the model field is called `date_subscribed`?

```suggestion
        fields = ['id', 'server_storage', 'subdomain', 'onboarding_state', 'date_subscribed']
        read_only_fields = ['id', 'subdomain', 'date_subscribed']
```

Reply:
Whoops no, that's me realising at the last moment that I forgot the API field, mixing up the names, and forgetting to test it :/ Fixed now.